python/lsst/analysis/tools/atools/refCatMatchPlots.py

```diff
@@ -642,3 +644,108 @@ def setDefaults(self):
+    target and reference catalog coordinates.
+    """
+
+    parameterizedBand = Field[bool](
+        doc="Does this AnalysisTool support band as a name parameter", default=False
+    )
+    AB2 = Field[float](
+        doc="Value by which no more than ABF1% of measurements shall deviate from the mean.",
+        default=20,
+    )
+
+    def setDefaults(self):
+        super().setDefaults()
+
+        self.parameterizedBand = False
+        self.prep.selectors.flagSelector = VisitPlotFlagSelector()
+        self.prep.selectors.starSelector = StarSelector()
+        self.prep.selectors.starSelector.vectorKey = "extendedness"
+
+        # Calculate difference in RA
+        self.process.buildActions.astromDiffRA = ConvertUnits(
+            buildAction=SubtractVector, inUnit="degree", outUnit="milliarcsecond"
+        )
+        self.process.buildActions.astromDiffRA.buildAction.actionA = RAcosDec(
+            raKey="coord_ra", decKey="coord_dec"
+        )
+        self.process.buildActions.astromDiffRA.buildAction.actionB = RAcosDec(raKey="r_ra", decKey="r_dec")
+        # Calculate difference in Dec
+        self.process.buildActions.astromDiffDec = ConvertUnits(
+            buildAction=SubtractVector, inUnit="degree", outUnit="milliarcsecond"
+        )
+        self.process.buildActions.astromDiffDec.buildAction.actionA = LoadVector(vectorKey="dec")
+        self.process.buildActions.astromDiffDec.buildAction.actionB = LoadVector(vectorKey="r_dec")
+
+        # Calculate total difference (using astropy)
+        self.process.buildActions.astromDiffTot = AngularSeparation(
+            raKey_A="coord_ra",
+            decKey_A="coord_dec",
+            raKey_B="r_ra",
+            decKey_B="r_dec",
+        )
+
+        self.process.buildActions.mags = ConvertFluxToMag()
+        self.process.buildActions.mags.vectorKey = "psfFlux"
+
+        # Filter down to only objects with mag 17-21.5
+        self.process.filterActions.brightStarsRA = DownselectVector(vectorKey="astromDiffRA")
+        self.process.filterActions.brightStarsRA.selector = RangeSelector(
+            vectorKey="mags", minimum=17, maximum=21.5
+        )
+
+        self.process.filterActions.brightStarsDec = DownselectVector(vectorKey="astromDiffDec")
+        self.process.filterActions.brightStarsDec.selector = RangeSelector(
+            vectorKey="mags", minimum=17, maximum=21.5
+        )
+
+        self.process.filterActions.brightStarsTot = DownselectVector(vectorKey="astromDiffTot")
+        self.process.filterActions.brightStarsTot.selector = RangeSelector(
+            vectorKey="mags", minimum=17, maximum=21.5
+        )
+
+        # Calculate median and sigmaMad
+        self.process.calculateActions.AB1_RA = RmsAction(vectorKey="brightStarsRA")
+        self.process.calculateActions.ABF1_RA = FracThreshold(
+            vectorKey="brightStarsRA", threshold=self.AB2, op="gt", relative_to_median=True, percent=True
+        )
```

Comment:
As agreed, this should go in a `finalize` block so that any runtime configs are picked-up (applies to the next few lines that use `self.AB2`, too).

Separately...looking at FracThreshold - doesn't it only return the fraction literally `gt` the threshold (so a large _negative_ separation will not be included in the fraction)? Since RA and Dec separation can have +ve and -ve values, wouldn't we want the fraction of abs(separation) > threshold?

Reply:
Yes, definitely. I'm annoyed at myself for missing this, as I already dealt with this on another metric! I think I have to add an option to `FracThreshold` to take the absolute value.